rpc_jobs/rpc_aio.yml

```diff
@@ -169,6 +169,7 @@
 
     dsl: |
       // CIT Slave node
+      currentBuild.result = 'SUCCESS'
```

Comment:
I see what you mean. Errors in delPubCloudSlave won't be caught because the try block is within pubcloud.runonpubcloud. That could be fixed by using a try/catch within the current runonpubcloud finally block. That could log and squash the exception so that teardown failures don't cause builds to fail, however thats an existing bug and should probably be fixed separately.